airdrop/serverless.yml

```diff
@@ -102,6 +102,26 @@ package:
     - "!node_modules/**"
 
 functions:
+  process_eligibility:
```

Comment:
Lambda timeout is set to 30 sec. Are we good with this timeout config?

Reply:
No, will update this.